virttest/utils_test/libvirt.py

```diff
@@ -2081,15 +2081,9 @@ def set_vm_disk(vm, params, tmp_dir=None, test=None):
                          emulated_image=emu_image,
                          image_size=image_size)
             # Get volume name
-            cmd_result = virsh.vol_list(pool_name)
-            try:
-                vol_name = re.findall(r"(\S+)\ +(\S+)[\ +\n]",
-                                      str(cmd_result.stdout))[1][0]
-            except IndexError:
-                raise exceptions.TestError("Fail to get volume name in "
-                                           "pool %s" % pool_name)
-            emulated_path = virsh.vol_path(vol_name, pool_name,
-                                           debug=True).stdout.strip()
+            vols = get_vol_list(pool_name)
+            vol_name = vols.keys()[0]
```

Comment:
The logic here is: if vol is None, then get_vol_list() will raise TestError inside, unless vol_check is False, but here we use the default value, and it will never trigger TypeError